openmetadata-spec/src/main/resources/json/schema/entity/services/connections/testConnectionDefinition.json

```diff
@@ -20,10 +20,19 @@
           "description": "What is the goal of the step",
           "type": "string"
         },
+        "errorMessage": {
+          "description": "In case of error this message should be displayed on UI",
+          "type": "string"
+        },
         "mandatory": {
           "description": "Is this step mandatory to be passed?",
           "type": "boolean",
           "default": true
+        },
+        "shortCircuit": {
+          "description": "Is this step to check the access to data source, if data source is not reachable stop further processing",
```

Comment:
Might not always be because of access to the data source. In further iterations of the logic, we might want to kill a test if, for example, we cannot list tables. I'd just define this as a show stopper step that is important enough to break the process